modules/swagger-core/src/main/java/io/swagger/util/PropertyDeserializer.java

```diff
@@ -88,6 +101,19 @@ Property propertyFromNode(JsonNode node) {
                 if (items != null) {
                     return new MapProperty(items).description(description);
                 }
+            } else {
+              detailNode = node.get("properties");
+              Map<String, Property> properties = new HashMap<String, Property>();
+              if(detailNode != null){
+                  for(Iterator<Map.Entry<String,JsonNode>> iter = detailNode.fields(); iter.hasNext();){
+                      Map.Entry<String,JsonNode> field = iter.next();
+                      if(!isReserved(field.getKey())){
```

Comment:
I am not sure it is necessary to check for the reserved property names here, some input would be welcome